boards/FSM/Test/Src/Test_StateMachine.cpp

```diff
@@ -83,12 +85,16 @@ class FsmStateMachineTest : public testing::Test
         rgb_led_sequence = App_SharedRgbLedSequence_Create(
             turn_on_red_led, turn_on_green_led, turn_on_blue_led);
 
+        brake_actuation_status =
```

Comment:
Kind of a nit, but would be nice to place this block of code about rgb_led_sequence to preserve the ordering

Reply:
fixed